frontend/src/utils/index.js

```diff
@@ -777,6 +778,44 @@ export function expiringWorkerGroupsForShoot (shootWorkerGroups, shootCloudProfi
   })
 }
 
+export function convertDuplicateSelectedItemsToPlaceholders (items, sort = true, valuePropertyName) {
```

Comment:
The optional parameter should be last. Otherwise the default makes not much sense.